src/common/form/fields/DateField.js

```diff
@@ -51,13 +52,20 @@ class UntranslatedDateField extends React.Component {
 
   render() {
     const {
+      end = false,
       locale,
       label,
       placeholder,
       value,
     } = this.props;
     const { isOpen } = this.state;
     const date = value || moment().toDate();
+    let valueOutput;
+
+    valueOutput = value ? moment(value).format('L') : null;
```

Comment:
This is not a neat solution, this should move into a handler function instead of being as part of render

Reply:
Dumb component actually need to be dumb, those calculation should be perform where this component get used.

TLDR: Those changes should not be placed here. This is a generic component